applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -183,6 +185,18 @@ static int get_input_report_idx(uint8_t report_id)
 	return -ENOENT;
 }
 
+static int get_output_report_size(uint8_t report_id)
```

Comment:
In most usecases in nRF Desktop report size does not include 1 byte of report ID. I think we should keep it consistent here too.

Reply:
Also maybe we could return size_t and include assert in this function if we want to limit supported range of report IDs?